src/openPR.ts

```diff
@@ -11,6 +11,7 @@ const openPR = async ({ owner, repo }: InputFields, username: string, branchName
         base: 'main',
         head: `refs/heads/${branchName}`,
         title: `@${username}'s Work: ${now}`,
+        draft: true,
```

Comment:
Interesting that the docs don't provide this type. Let's try it. Feel free to run it in [daily-update-action repo](https://github.com/inkblotty/daily-update-action/actions) to test.